Index: src/main/kotlin/no/cloudberries/candidatematch/Main.kt
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>package no.cloudberries.candidatematch\n\nimport org.springframework.boot.SpringApplication\nimport org.springframework.boot.autoconfigure.SpringBootApplication\nimport org.springframework.data.web.config.EnableSpringDataWebSupport\nimport org.springframework.data.web.config.EnableSpringDataWebSupport.PageSerializationMode\nimport org.springframework.scheduling.annotation.EnableScheduling\n\n@EnableSpringDataWebSupport(pageSerializationMode = PageSerializationMode.VIA_DTO)\n@SpringBootApplication\n@EnableScheduling\nclass Main\n\nfun main(args: Array<String>) {\n    SpringApplication.run(Main::class.java, *args)\n}\n
===================================================================
diff --git a/src/main/kotlin/no/cloudberries/candidatematch/Main.kt b/src/main/kotlin/no/cloudberries/candidatematch/Main.kt
--- a/src/main/kotlin/no/cloudberries/candidatematch/Main.kt	(revision 133de0a75a2a5db2d5cc7f6719f99dae65f5f276)
+++ b/src/main/kotlin/no/cloudberries/candidatematch/Main.kt	(date 1758654659965)
@@ -2,15 +2,13 @@
 
 import org.springframework.boot.SpringApplication
 import org.springframework.boot.autoconfigure.SpringBootApplication
-import org.springframework.data.web.config.EnableSpringDataWebSupport
-import org.springframework.data.web.config.EnableSpringDataWebSupport.PageSerializationMode
 import org.springframework.scheduling.annotation.EnableScheduling
 
-@EnableSpringDataWebSupport(pageSerializationMode = PageSerializationMode.VIA_DTO)
 @SpringBootApplication
 @EnableScheduling
 class Main
 
 fun main(args: Array<String>) {
-    SpringApplication.run(Main::class.java, *args)
+    val app = SpringApplication(Main::class.java)
+    app.run(*args)
 }
